presto-docs/src/main/sphinx/installation/deploy-docker.rst

```diff
@@ -0,0 +1,116 @@
+
+   ``colima start``
+
+4. To verify your local setup, run the following command:
+
+   ``docker run hello-world``
+
+   If you see a response similar to the following, you are ready.
+
+   ``Hello from Docker!`` 
+   ``This message shows that your installation appears to be working correctly.``
+
+Installing and Running the Presto Docker container
+==================================================
+
+1. Run the following command to download the prestodb-sandbox container from `DockerHub <https://hub.docker.com/r/ahanaio/prestodb-sandbox>`_: 
+
+   ``docker pull ahanaio/prestodb-sandbox``
+
+   Downloading the container may take a few minutes. When the download completes, go on to the next step.
+
+2. To start the Presto server in the Docker container, run the following command:
+
+   ``docker run -p 8080:8080 --name presto ahanaio/prestodb-sandbox``
+
+   This command also assigns the name ``presto`` for ``ahanaio/prestodb-sandbox``.
+
+  The Presto server begins logging startup information in the terminal window. Once you see a response similar to the following, the Presto server is running in the Docker container.
+
+   ``======== SERVER STARTED ========``
+
+3. To start the Presto CLI (command-line interface), open a new terminal window and run the following command:
+
+   ``docker exec -it presto presto-cli``
+
+   The presto prompt is shown:
+
+   ``presto>``
+
+   If the ``docker exec`` command returns an error similar to the following:
+   
+   ::
+    
+    "docker exec" requires at least 2 arguments.
+    See 'docker exec --help'.
+    Usage:  docker exec [OPTIONS] CONTAINER COMMAND [ARG...]
+    Execute a command in a running container``
+
+   the Presto server has not completed starting. Wait a little and try again, until the ``presto>`` prompt is shown.
+
+4. To verify that your Presto server and Presto CLI are communicating together, use the Presto CLI to query your local Presto server. Run the following command at the ``presto>`` prompt:
+
+   ``show catalogs;``
+
+   The response should be similar to the following:
+
+   ::
+
+    presto> show catalogs;
+    Catalog
+    ---------
+    jmx
+    memory
+    system
+    tpcds
+    tpch
+    (5 rows)
+    
+    Query 20230614_181140_00000_uutw6, FINISHED, 1 node
+    Splits: 19 total, 19 done (100.00%)
+    [Latency: client-side: 0:13, server-side: 0:11] [0 rows, 0B] [0 rows/s, 0B/s]
+    
+    presto>
+
+5. To use the Presto CLI in your local Docker image to connect to a remote Presto server, run the following command: 
+
+   ``docker exec -it presto presto-cli --server Presto-endpoint-URL --user username --password``
+
+   You will be prompted to enter the password for the Presto user, then the ``presto>`` prompt is shown.
+
+6. To verify your Presto server and Presto CLI are communicating together, use the Presto CLI to query your remote Presto server. 
+Run the ``show catalogs;`` command at the ``presto>`` prompt and the result should be similar to step 4.
+
```

Comment:
is it possible to also provide an extra section to explain how to clean up the presto container, like: `docker stop presto`. `docker rm presto`?

Reply:
I'd thought about writing that, and chose to leave it out because I was trying to keep the steps as short as possible. But  because you're asking I've reconsidered, and I think I'll include it. Thanks!